lib/commands/test.js

```diff
@@ -60,6 +55,7 @@ var command = {
           temp.cleanup(function(err) {
             // Ignore cleanup errors.
             done.apply(null, args);
+            process.exit(0);
```

Comment:
Haven't tested it yet, but I believe this will kill a running console if the `test` command is run.